src/diffusers/__init__.py

```diff
@@ -70,7 +70,9 @@
 except OptionalDependencyNotAvailable:
     from .utils import dummy_pt_objects  # noqa F403
 
-    _import_structure["utils.dummy_pt_objects"] = [name for name in dir(dummy_pt_objects) if not name.startswith("_")]
+    _import_structure["utils.dummy_pt_objects"] = [
```

Comment:
were these changes made intentionally/manually? if so, it'll have to be reverted. we use [ruff](https://github.com/astral-sh/ruff) for formatting using [this](https://github.com/huggingface/diffusers/blob/main/pyproject.toml) configuration. you can automatically fix most of these by running `make style` in the diffusers root directory, or manually reverting